man/systemd-nspawn.xml

```diff
@@ -1091,6 +1091,17 @@
     </example>
 
     <example>
+      <title>Install the OpenSUSE Tumbleweed rolling distribution</title>
+
+      <programlisting># zypper --root=/var/lib/machines/tumbleweed ar \
+      https://download.opensuse.org/tumbleweed/repo/oss tumbleweed
+# zypper --root=/var/lib/machines/tumbleweed refresh
+# zypper --root=/var/lib/machines/tumbleweed install --no-recommends systemd
```

Comment:
The package list should be `systemd shadow zypper openSUSE-release vim`, I believe.